index.js

```diff
@@ -0,0 +1,112 @@
+const express = require('express')
+const server = express()
+const cors = require('cors')
+const bcrypt = require('bcryptjs')
+const knex = require('knex')
+const knexConfig = require('./knexfile.js')
+const session = require('express-session')
+const KnexSessionStore = require('connect-session-knex')(session)
+
+const db = knex(knexConfig.development)
+
+const sessionConfig = {
```

Comment:
great job configuring your sessions and adding the commenting to better help you understand what is doing what.